src/View/FirstTimeWelcomeWindow.py

```diff
@@ -0,0 +1,222 @@
+        self.save_dir_button = QPushButton()
+        self.save_dir_button.setObjectName("SaveDirButton")
+        self.save_dir_button.setSizePolicy(
+            QSizePolicy(QSizePolicy.MinimumExpanding, QSizePolicy.MinimumExpanding))
+        self.save_dir_button.resize(self.save_dir_button.sizeHint().width(),
+                                    self.save_dir_button.sizeHint().height())
+        self.save_dir_button.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
+        self.save_dir_button.clicked.connect(self.save_default_dir)
+        self.save_dir_button.setProperty("QPushButtonClass", "success-button")
+        self.first_time_window_configure_actions_horizontal_box.addWidget(self.save_dir_button)
+
+        # Create a widget to house all of the actions button for first-time window
+        self.first_time_window_configure_actions_widget = QWidget()
+        self.first_time_window_configure_actions_widget.setLayout(
+            self.first_time_window_configure_actions_horizontal_box)
+        self.window_vertical_layout_box.addWidget(self.first_time_window_configure_actions_widget)
+
+        self.first_time_window_instance_central_widget = QWidget()
+        self.first_time_window_instance_central_widget.setLayout(self.window_vertical_layout_box)
+        first_time_welcome_window_instance.setCentralWidget(self.first_time_window_instance_central_widget)
+
+        # Set the current stylesheet to the instance and connect it back to the caller through slot
+        _stylesheet = open(resource_path(self.stylesheet_path)).read()
+        first_time_welcome_window_instance.setStyleSheet(_stylesheet)
+        self.retranslate_ui(first_time_welcome_window_instance)
+        QtCore.QMetaObject.connectSlotsByName(first_time_welcome_window_instance)
+
+    # this function inserts all the text in the first time page
+    def retranslate_ui(self, first_time_welcome_window_instance):
+        _translate = QtCore.QCoreApplication.translate
+        first_time_welcome_window_instance.setWindowTitle(
+            _translate("FirstTimeWelcomeWindowInstance", "OnkoDICOM - First-time User Setting"))
+        self.first_time_welcome_message_label.setText(_translate("FirstTimeWelcomeWindowInstance", "Welcome to OnkoDICOM!"))
+        self.first_time_welcome_message_slogan.setText(_translate("FirstTimeWelcomeWindowInstance",
+                                                      "OnkoDICOM - the solution for producing data for analysis from your oncology plans and scans."))
+        self.first_time_welcome_default_dir_prompt.setText(_translate("FirstTimeWelcomeWindowInstance","Choose the path of the default directory containing all DICOM files:"))
+        self.first_time_welcome_input_box.setPlaceholderText(_translate("FirstTimeWelcomeWindowInstance",
+                                                                            "Enter DICOM Files Path (For example, C:\path\\to\your\DICOM\Files)"))
```

Comment:
Two slashes between path and to.  Is that intentional?

Reply:
yes, this is intentional. if the second slash is removed, the "t" in "to" will disappear.
![image](https://user-images.githubusercontent.com/62133130/124376541-c6298580-dcea-11eb-9c0a-4fc48706a70f.png)

Reply:
Ah... right. Escaping the escape character for tab...